kano_world/share_helpers.py

```diff
@@ -0,0 +1,31 @@
+# share_helpers.py
+#
+# Copyright (C) 2016 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+#
+from kano.utils.shell import run_cmd
+from .functions import login_using_token
+from .share import upload_share
+
+
+def login_and_share(file_path, title, app_name):
+    """ Make a share to Kano World. If the user is not logged in, present the
+    register/login window to the user.
+    :param file_path: Code file to be uploaded to Kano World
+    :type str:
```

Comment:
The variable name is missing here.

I'd also format the comment a little to make it a bit more readable (but that's just my personal preference):

https://github.com/KanoComputing/kano-init/blob/master/kano_init/user.py#L34